ShapesProject/Project1/ValidateInput.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+#include <iostream>
+
```

Comment:
Hmmm this is a programming style I hadn't seen, I guess it's not incorrect to make a .h and .cpp files just to declare a couple of functions... I'm still confused though